tests/UserAgentsTest/FullTest.php

```diff
@@ -135,46 +130,7 @@ public static function tearDownAfterClass() : void
 
     public function userAgentDataProvider() : array
     {
-        $data = [];
-
-        $checks          = [];
-        $sourceDirectory = __DIR__ . '/../issues/';
-        $iterator        = new \RecursiveDirectoryIterator($sourceDirectory);
-
-        foreach (new \RecursiveIteratorIterator($iterator) as $file) {
-            /** @var $file \SplFileInfo */
-            if (!$file->isFile() || 'php' !== $file->getExtension()) {
-                continue;
-            }
-
-            $tests = require $file->getPathname();
-
-            foreach ($tests as $key => $test) {
-                if (isset($data[$key])) {
-                    throw new \RuntimeException('Test data is duplicated for key "' . $key . '"');
```

Comment:
This method used to throw an exception if there was an error, which would halt test execution (I believe).

It seems that now, nothing will happen, and the tests will just continue as normal no?

Should the line added below be checking the `$errors` value that is returned, and throw an exception if that's true?  I think it should probably also pass in a logger that the log messages could be retrieved from, so that we can output a friendly error message if any of these situations are encountered.